lib/ansible/modules/network/eos/eos_config.py

```diff
@@ -279,7 +279,7 @@
 
 
 def get_candidate(module):
-    candidate = NetworkConfig(indent=2)
+    candidate = NetworkConfig(indent=3)
```

Comment:
Indent was changed from 3 to 2 as part of this PR https://github.com/ansible/ansible/pull/27686. This change again reverts it back to 3. Will this change result issue #25858 which was fixed with PR #27686